core/app_config/AppConfigBase.cpp

```diff
@@ -239,6 +253,8 @@ void AppConfigBase::LoadIncludeConfig(Json::Value& confJson) {
 void AppConfigBase::LoadAppConfig(const std::string& ilogtailConfigFile) {
     std::string processExecutionDir = GetProcessExecutionDir();
     mDockerFilePathConfig = processExecutionDir + STRING_FLAG(ilogtail_docker_file_path_config);
+    LoadEnvTags();
```

Comment:
Please config the sequence betweenn overwrite_community_edition_flags and LoadAppConfig